src/common/constants/colors.ts

```diff
@@ -4,4 +4,8 @@ export const colors = {
   primaryOne: "#004A93",
   primaryTwo: "#3677C7",
   lightGrey: "#eee",
+  inputBackground: 'rgba(13,13,13,19%)',
+  inputPlaceholder: 'rgba(255,255,255,27%)',
+  inputText: 'rgba(255,255,255,50%)',
+  inputCaretColor: 'rgba(255,255,255,60%)'
```

Comment:
The names here depict coupling with a particular component, we can use more generic names like: `grey-20` etc.